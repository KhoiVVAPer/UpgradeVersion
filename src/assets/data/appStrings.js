/* eslint-disable */

const APP_STRINGS = {
  product_warning_str: 'Unfortunately, the desired product is not part of our current product range anymore. Accessories, cleaning products and operating instructions are still available.',
  watchAgainTxt: 'Here you can find the introduction and explaination of the different app functions.',
};

export default APP_STRINGS;
