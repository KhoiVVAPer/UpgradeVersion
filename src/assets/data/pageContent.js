/* eslint-disable */
const headlineText = {
  "id": "content-13",
  "type": "content",
  "config": '{"layout":"manualcontent_headlinetext","values":[{"headline":"Headline","text":"<b>Lorem ipsum dolor sit amet</b>, consetetur sadipscing elitr, <a href=\\"https://www.google.com/\\">sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd</a> gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, <i>no sea takimata sanctus</i> est Lorem ipsum dolor sit amet."}]}'
};

const textContent = {
  "id": "content-11",
  "type": "content",
  "config": '{"layout":"manualcontent_text","values":[{"text":"<p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.</p>"}]}'
};

const ribbon = {
  "id": "content-14",
  "type": "content",
  "config": '{"layout":"manualcontent_ribbon","values":[{"headline":"Headline","text":"<b>Lorem ipsum dolor sit amet</b>, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, <i>no sea takimata sanctus</i> est Lorem ipsum dolor sit amet."}]}'
};

const imageHeadlineText = {
  "id": "content-15",
  "type": "content",
  "config": '{"layout":"manualcontent_imageheadlinetext","values":[{"headline":"Headline","text":"Lorem <b>ipsum</b> dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.","image":"https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg"}]}'
};

const imageHeadlineTextFiftyFifty = {
  "id": "content-16",
  "type": "content",
  "config": '{"layout":"manualcontent_imageheadlinetextfiftyfifty","values":[{"headline":"Headline","text":"<p>Lorem ipsum dolor sit amet, <b>consetetur sadipscing</b> elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo <ul><li>dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.</li> <li>Lorem ipsum dolor sit amet, consetetur sadipscing elitr,</li></ul> sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.</p>","imageRight":"1","image":"https://s1.kaercher-media.com/products/13171300/main/1/d3.jpg"}]}'
};

const fullWidthImage = {
  "id": "content-17",
  "type": "content",
  "config": '{"layout": "manualcontent_image","values": [{"image": "https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg"}]}'
};

const imageHeadlineTextTwoCol = {
  "id": "content-18",
  "type": "content",
  "config": '{"layout":"manualcontent_imageheadlinetexttwocol","values": [{"image_left": "https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg","headline_left": "Headline left","text_left": "<p>Maecenas nec odio et ante tincidunt tempus. Donec vitae sapien ut libero venenatis faucibus. Nullam quis ante. Etiam sit amet orci eget eros faucibus tincidunt. Duis leo. Sed fringilla mauris sit amet nibh. Donec sodales sagittis magna. Sed consequat, leo eget bibendum sodales, augue velit cursus nunc,</p>","image_right": "https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg","headline_right": "Headline right","text_right": "<p>Maecenas nec odio et ante tincidunt tempus. Donec vitae sapien ut libero venenatis faucibus. Nullam quis ante. Etiam sit amet orci eget eros faucibus tincidunt. Duis leo. Sed fringilla mauris sit amet nibh. Donec sodales sagittis magna.</p>"}]}'
};

const imageHeadlineTextThreeCol = {
  "id": "content-19",
  "type": "content",
  "config": '{"layout":"manualcontent_imageheadlinetextthreecol","values": [{"image_left": "https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg","headline_left": "Headline left one","text_left": "<p>Maecenas nec odio et ante tincidunt tempus. Donec vitae sapien ut libero venenatis faucibus. Nullam quis ante. Etiam sit amet orci eget eros faucibus tincidunt. Duis leo. Sed fringilla mauris sit amet nibh. Donec sodales sagittis magna. Sed consequat, leo eget bibendum sodales, augue velit cursus nunc,</p>","image_middle": "https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg","headline_middle": "Headline middle column","text_middle": "<p>Maecenas nec odio et ante tincidunt tempus. Donec vitae sapien ut libero venenatis faucibus. Nullam quis ante. Etiam sit amet orci eget eros faucibus tincidunt. Duis leo. Sed fringilla mauris sit amet nibh. Donec sodales sagittis magna. Sed consequat, leo eget bibendum sodales, augue velit cursus nunc,</p>","image_right": "https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg","headline_right": "Headline right side","text_right": "<p>Maecenas nec odio et ante tincidunt tempus. Donec vitae sapien ut libero venenatis faucibus. Nullam quis ante. Etiam sit amet orci eget eros faucibus tincidunt. Duis leo. Sed fringilla mauris sit amet nibh. Donec sodales sagittis magna. Sed consequat, leo eget bibendum sodales, augue velit cursus nunc,</p>"}]}'
};

const imageHeadlineTextFourCol = {
  "id": "content-167",
  "type": "content",
  "config": '{"layout":"manualcontent_imageheadlinetextfourcol","values": [{"image_first": "https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg","headline_first": "Headline first","text_first": "<p>Maecenas nec odio et ante tincidunt tempus. Donec vitae sapien ut libero venenatis faucibus. Nullam quis ante. Etiam sit amet orci eget eros faucibus tincidunt. Duis leo. Sed fringilla mauris sit amet nibh. Donec sodales sagittis magna.</p>","image_second": "https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg","headline_second": "Headline second","text_second": "<p>Maecenas nec odio et ante tincidunt tempus. Donec vitae sapien ut libero venenatis faucibus. Nullam quis ante. Etiam sit amet orci eget eros faucibus tincidunt. Duis leo. Sed fringilla mauris sit amet nibh. Donec sodales sagittis magna.</p>","image_third": "https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg","headline_third": "Headline third","text_third": "<p>Maecenas nec odio et ante tincidunt tempus. Donec vitae sapien ut libero venenatis faucibus. Nullam quis ante. Etiam sit amet orci eget eros faucibus tincidunt. Duis leo. Sed fringilla mauris sit amet nibh. Donec sodales sagittis magna.</p>","image_fourth": "https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg","headline_fourth": "Headline fourth","text_fourth": "<p>Maecenas nec odio et ante tincidunt tempus. Donec vitae sapien ut libero venenatis faucibus. Nullam quis ante. Etiam sit amet orci eget eros faucibus tincidunt. Duis leo. Sed fringilla mauris sit amet nibh. Donec sodales sagittis magna.</p>"}]}'
};

const horizontalRule = {
  "id": "content-rule",
  "type": "content",
  "config": '{"layout":"manualcontent_horizoantalrule"}'
};

const anchors = {
  "id": "anchor",
  "type": "content",
  "config": '{"layout":"manualcontent_anchor","values":[{"targetId":"content-13","title":"Headline Text"},{"targetId":"content-15","title":"Image Headline"},{"targetId":"content-16","title":"Left Image Headline"},{"targetId":"content-167","title":"Four Column"},{"targetId":"content-17","title":"Full Image"},{"targetId":"content-18","title":"Two Column"},{"targetId":"content-19","title":"Three Column"},{"targetId":"table","title":"Table"}]}'
}

const table = {
  "id": "table",
  "type": "content",
  "config": '{"layout":"manualcontent_table","values":[[[{"value":"<b>Column1</b>","style":""},{"value":"<b>Column2</b>","style":""}],[{"value":"<b>Content</b><ul><li>item</li><li>item</li><li>item</li></ul>","style":""},{"value":"<b>Content</b><ul><li>item</li><li>item</li><li>item</li></ul>","style":""}],[{"value":"<b>Content</b><ul><li>item</li><li>item</li><li>item</li></ul>","style":""},{"value":"<b>Content</b><ul><li>item</li><li>item</li><li>item</li></ul>","style":""}],[{"value":"<b>Content</b><ul><li>item</li><li>item</li><li>item</li></ul>","style":""},{"value":"<b>Content</b><ul><li>item</li><li>item</li><li>item</li></ul>","style":""}]]]}'
}

const businessUnitEntry = {
  values: [{
    image_left: 'https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg',
    headline_left: '',
    button_link_left: '',
    button_text_left: '',
    image_right: 'https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg',
    headline_right: '',
    button_link_right: '',
    button_text_right: '',
  }]
}

const imageCollection = {
  values: [{
    imageBig: 'https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg',
    buttonlink_imageBig: '1000',
    headline_imageBig: 'Test',
    buttonlink_imageBig: '345345345',
    buttontext_imageBig: 'Test',
    headline_imageSmallTop: 'Test',
    imageSmallTop: 'https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg',
    link_imageSmallTop: 'Test',
    headline_imageSmallMiddle: 'Test',
    imageSmallMiddle: 'https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg',
    link_imageSmallMiddle: 'Test',
    headline_imageSmallBottom: 'Test',
    imageSmallBottom: 'https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg',
    link_imageSmallBottom: 'Test',
  }]
}

export {
  headlineText,
  textContent,
  ribbon,
  imageHeadlineText,
  imageHeadlineTextFiftyFifty,
  fullWidthImage,
  imageHeadlineTextTwoCol,
  imageHeadlineTextThreeCol,
  imageHeadlineTextFourCol,
  horizontalRule,
  anchors,
  table,
  businessUnitEntry,
  imageCollection
};
