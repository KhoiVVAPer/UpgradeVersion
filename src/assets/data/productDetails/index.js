import * as accessoriesData from './accessories';
import applicationsArr from './applicationsArr';
import detergentWarningsArr from './detergentWarningsArr';
import anchorArr from './anchorArr';

export {
  accessoriesData,
  applicationsArr,
  detergentWarningsArr,
  anchorArr
};
