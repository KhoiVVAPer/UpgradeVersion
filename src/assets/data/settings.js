/* eslint-disable indent, max-len */
export default {
    settingsPrivacyTxt: `1. Besuch der Kärcher-Websites generell

Beim Besuch der Kärcher-Websites übermitteln Sie (aus technischer Notwendigkeit) über Ihren Internetbrowser Daten an unseren Webserver. Folgende Daten werden während einer laufenden Verbindung zur Kommunikation zwischen Ihrem Internet­browser und unserem Webserver aufgezeichnet:

Datum und Uhrzeit der Anforderung Name der angeforderten Datei Seite, von der aus die Datei angefordert wurde Zugriffsstatus (Datei übertragen, Datei nicht gefunden, etc.) verwendeter Webbrowser und verwendetes Betriebssystem vollständige IP-Adresse des anfordernden Rechners übertragene Datenmenge Aus Gründen der technischen Sicherheit, insbesondere zur Abwehr von Angriffs­ver­suchen auf unseren Webserver, werden diese Daten von uns kurzzeitig gespei­chert. Ein Rückschluss auf einzelne Personen ist uns anhand dieser Daten nicht möglich.

Rechtsgrundlage der Speicherung ist § 15 TMG und Artikel 6 lit. f) DSGVO.

Weitergehende personenbezogene Informationen werden nur erfasst, wenn Sie diese freiwillig zur Verfügung stellen, etwa im Rahmen einer Anfrage oder Registrie­rung. Die von Ihnen zur Verfügung gestellten personenbezogenen Daten verwendet Kärcher je nach betroffenem Bereich zur Beantwortung Ihrer Anfragen, zur Abwick­lung Ihrer Bestellung sowie zum Zweck der technischen Administration der Web­seiten. Im Einzelnen folgt die Nutzung in den jeweiligen Bereichen wie folgt:

2. Onlineshop

Besuchen Sie unseren Onlineshop speichern wir zur Erfüllung des zwischen Ihnen und Kärcher geschlossenen Vertrages bzw. zur Durchführung vorvertraglicher Maß­nahmen nach Artikel 6 lit. b) DSGVO folgende Informationen:

a) Bestellung ohne Einrichtung eines Kundenkontos
Bei einer Bestellung im Rahmen des Onlineshops werden mittels Pflichtfelder alle Daten abgefragt, die zur Ausführung und Abwicklung notwendig sind: Ihr vollstän­diger Name, Ihre E-Mail-Adresse, Ihre Anschrift (Rechnungs- und ggf. abweichende Lieferanschrift). Ihre Daten werden nur zur Abwicklung Ihrer Bestellung verwendet.`,

imprintText: `Alfred Kärcher SE & Co. KG

Alfred-Kärcher-Strasse 28-40

71364 Winnenden / Germany
P.O. Box 160
71349 Winnenden / Germany

Tel. +49 7195 / 14-0
Fax +49 7195 / 14-2212
E-mail: info@karcher.com

Alfred Kärcher SE & Co. Kommanditgesellschaft
(private limited partnership)
Registered office: Winnenden
Court of Registration: Stuttgart, HRA 260 169

General partner:
Kärcher SE, registered office: Winnenden,
Court of Registration: Stuttgart, HRB 765 434

VAT ID no: DE 147223775
ILN: 4039784000001

Board of Management:
Hartmut Jenner (Chairman),
Markus Asch (Deputy Chairman),
Dieter Grajer, Christian May

Chairman of the Supervisory Board:
Johannes Kärcher`
};
