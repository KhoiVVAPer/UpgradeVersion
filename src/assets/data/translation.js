/* eslint-disable max-len */
export default [{
  export_send: 'Export/Send',
  accessories: 'Accessories',
  add_comment: 'Add Comment',
  add_to_favorites: 'Add to favorites',
  added_to_favorites: 'Added to favorites',
  all: 'All',
  attention: 'Attention:',
  back: 'Back',
  back_to_start: 'Back to start',
  cancel: 'Cancel',
  programme: 'Programme',
  catalog: 'Catalog',
  category: 'Category',
  change: 'Change',
  change_the_country: 'Change the country',
  change_the_language: 'Change the language',
  cleaning_agents: 'Cleaning agents',
  click_logo: 'Click on the logo to return to the start screen and to the Home & Garden and Professional selection.',
  compare: 'Compare',
  continue: 'Continue',
  country_and_language_settings: 'Country and language settings',
  countryandlanguage_selection: 'Please choose your country and language',
  create_new_folder: 'Create new folder',
  data_can_not_be_loaded: 'Data can not be loaded as you are offline',
  data_load_error: 'Sorry, Data could not be loaded at the moment',
  datasheet: 'Datasheet',
  delete: 'Delete',
  description: 'Description',
  do_you_want_to_change_settings: 'Are you sure you want to change language?',
  download: 'Download',
  downloading_starts_now: 'Download starts now',
  downloads: 'Downloads',
  duplicate: 'Duplicate',
  edit: 'Edit',
  enter_new_foldername: 'Enter new folder name',
  enter_the_foldername: 'Please enter the folder name',
  equipment: 'Equipment',
  error: 'Error',
  error_404: 'Error 404',
  export: 'Export',
  export_select: 'Export / Select',
  favorite: 'Favorite',
  favorites: 'Favorites',
  feature_and_benefits: 'Feature and Benefits',
  file_could_not_be_open: 'Sorry, File could not be open/downloaded',
  filter: 'Filter',
  service_offer_3: 'Full Service',
  here_stored_favorites: 'Here you can find your stored favorites and manage them.',
  home_garden: 'Home & Garden',
  imprint: 'Imprint',
  imprint_policy_text: 'Imprint policy text',
  info_and_version: 'Info & Version',
  service_offer_1: 'Inspect',
  service_offer_2: 'Maintain',
  item_added_to_comparison: 'The following item was added to comparison',
  last_download: 'Last download',
  launched_in: 'launched in',
  maintain: 'Maintain',
  manual: 'Manual',
  more: 'More',
  more_information: 'More information',
  my_favorites: 'My Favorites',
  new_folder: 'New Folder',
  next: 'Next',
  not_available: 'Sorry, this content is at the moment not available because you are offline.\nPlease try again later.',
  ok: 'Ok',
  order_number: 'Order number',
  play: 'Play',
  please_enter_foldername: 'Please enter the folder name',
  please_select: 'Please Select',
  please_select_country: 'Please select country',
  please_select_language: 'Please select language',
  price_in: 'Price in',
  privacy_policy_text: 'Privacy ploicy text',
  privacy_statement: 'Privacy Statement',
  product_comparison: 'Product comparison',
  product_groups: 'Product Groups',
  product_left: 'Products Left',
  professional: 'Professional',
  quit: 'Quit',
  remove: 'Remove',
  reset_filter: 'Reset Filter',
  results_for: 'Results for',
  save: 'Save',
  save_folder: 'Save folder',
  search_for_product_or_keyword: 'Search For Product or Keyword',
  select_country: 'Select country',
  select_language: 'Select language',
  services: 'Services',
  settings: 'Settings',
  settings_download_caption: 'Simply select which areas you would like to download',
  settings_introduction_caption: 'Here you can find the introduction and explaination of the different app functions.',
  show_more: 'Show more',
  show_more_accessories: 'Show More Accessories',
  show_more_detergents: 'Show more detergents',
  skip: 'Skip',
  sort_by_name: 'Sort by name',
  sparepartlist: 'Sparepartlist',
  start: 'Start',
  statusindicator: 'Statusindicator',
  stop: 'Stop',
  success: 'Success',
  technical_data: 'Technical data',
  thankyou_for_downloading: 'Thank you for downloading the Kärcher Programme App. We have prepared a few tips for you so that you can get the most out of our app.',
  use_mobile_data_fordownload: 'Use mobile data for download',
  version: 'Version',
  warning: 'Warning',
  watch_the_onboarding_again: 'Watch the onboarding again',
  welcome: 'Welcome',
  welcome_to_karcher: 'Welcome the Kärcher Programme 2020',
  safetydatasheet: 'Safty Datasheet',
  eudatasheet: 'EU Datasheet',
  detergent_warning: 'Detergent Warnings',
  applications: 'Applications',
  compatible_machines: 'Compatible Machines',
  actual_machines: 'Actual machines',
  old_machines: 'Old Machines',
  configurable_component: 'Configurable Components',
  compare_alert: 'No products are selected for comparison.',
  return_to_previous_page: 'Return to Previous page',
  // configuration_progress: 'Please wait. Configuration is in progress.',
  to_comparison: 'TO COMPARISON',
  product_removed_from_compare_alert: 'Product removed from compare',
  no_product_is_added_for_comparison_alert: 'No product is added for comparison',
  comments_is_saved: 'Comments is saved',
  comments_is_deleted: 'Comments is deleted',
  no_products_for_comparison_alert: 'No products are selected for comparison',
  user_offline_message: 'Sorry, this content is at the moment not available because you are offline. Please try again later',
  download_starts_now: 'DOWNLOAD STARTS NOW',
  nothing_is_selected_for_download: 'Nothing is selected for download',
  resume_download_alert: 'Offline downloading is incomplete. Do you want to resume it again?',
  offline_download_completed: 'Offline download completed',
  download_error_for_mobile_netowork__alert: 'Device is connected to mobile network. Download will start when connected to wifi network.',
  downloading_error_alert: 'Sorry, Some error occured while dowloading.',
  add_marking: 'ADD MARKING',
  delete_marking_warning: 'Do you want to delete marking?',
  marking_deleted: 'Marking deleted',
  marking_is_saved: 'Marking is saved',
  create_text_marking: 'CREATE TEXT MARKING',
  add_text_comment: 'ADD TEXT COMMENT',
  no_text_is_marked: 'No text is marked',
  please_add_the_comment_first: 'Please add the comment first',
  comment_date: 'Added on',
  no_favorites: 'No Favorites Available for this category',
  content_not_available: 'Sorry, this content at the moment not available because you are offline.',
  please_try_again: 'Please try again later.',
  no_data_available: 'No data available',
  save_new_folder: 'Save new folder',
  three_folder_validation_alert: 'We can only save upto three folder in tree',
  okay: 'OKAY',
  comment: 'Comment',
  delete_comment: 'Delete Comment',
  max_four_product_warning_alert: 'Maximum four products can be added',
  detergent_warnings_andsafety_rec: 'DETERGENT WARNINGS & SAFETY RECOMMENDATIONS',
  search_result: 'Search result',
  product_finder: 'Product Finder',
  load_more: 'Load more',
  result_for: 'result for',
  not_available_online: 'Sorry, this content is not availble at the moment',
  goto_settings: 'Goto settings',
  not_found: 'Not Found',
  onboarding_root_heading: 'Willkommen im Kärcher Programm',
  onboarding_root_description: 'Lernen Sie die wichtigsten Funktionen im Kärcher Programm kennen.',
  onboarding_logo_heading: 'Zurück zum Anfang',
  onboarding_logo_description: 'Klicken Sie auf das Kärcher-Logo, um zum Startbildschirm zurück zu gelangen. Sie kommen so auch zurück zu den Bereichen Home & Garden sowie Professional.',
  onboarding_scanner_heading: 'QR Code & Barcode Scan',
  onboarding_scanner_description: 'This feature helps to scan QR Code and Barcode of Kaercher product and based on the Order Number, application takes to the details for the Product.',
  onboarding_onlineOfflineIndicator_heading: 'Statusanzeige',
  onboarding_onlineOfflineIndicator_description: 'Über die Statusanzeige sehen Sie auf einen Blick, ob die Anwendung online oder offline ist.\n\nBitte beachten Sie, dass die Daten nur aktualisiert werden, wenn Sie online sind.',
  onboarding_favorite_heading: 'Favoriten',
  onboarding_favorite_description: 'Hier können Sie Ihre gespeicherten Favoriten finden und verwalten.\n\nEine Seite speichern Sie ganz einfach als Favorit, indem Sie das Sternsymbol anklicken.',
  onboarding_productCompare_heading: 'Produktvergleich',
  onboarding_productCompare_description1: 'Innerhalb unserer App haben Sie die Möglichkeit Produkte\nzu vergleichen. Klicken Sie einfach die folgende Option an:',
  onboarding_productCompare_description2: 'Anschließend finden Sie den Produktvergleich hier.',
  onboarding_moreInformationRibbon_heading: 'Mehr Informationen',
  onboarding_moreInformationRibbon_description: 'Über das Feld Mehr Informationen erhalten Sie weiterführende Inhalte zur jeweiligen Produktgruppe,\nz. B. zu verschiedenen Anwendungsmöglichkeiten.',
  onboarding_orderNumber_heading: 'Markieren und Kommentieren',
  onboarding_orderNumber_description: 'Sie können jeden Text farblich markieren. Außerdem können Sie auf allen Seiten Ihre individuellen Kommentare einfügen.',
  onboarding_export_heading: 'Seitenexport',
  onboarding_export_description: 'Sie können jede Seite exportieren und z. B. als PDF abspeichern und versenden.',
  onboarding_download_heading: 'Offlinenutzung',
  onboarding_download_description: 'Wenn Sie das Kärcher Programm auch offline nutzen möchten, haben Sie unter Einstellungen die Möglichkeit,\ndie Inhalte auf Ihr Endgerät herunterzuladen.\n\nDie Inhalte aktualisieren sich bei der nächsten Internetverbindung automatisch.',
  onboarding_changeLanguage_heading: 'Sprachenwechsel',
  onboarding_changeLanguage_description: 'Über Einstellungen können Sie die Sprache des Kärcher Programms nach Ihren Vorlieben wechseln.',
  onboarding_playOnboarding_heading: 'Noch einmal ansehen',
  onboarding_playOnboarding_description: 'Über Einstellungen können Sie sich diese Anleitung erneut anschauen, um die wichtigsten Funktionen des Kärcher Programms kennenzulernen.',
  finish: 'Finish',
  favorite_deleted: 'Favorite deleted',
  download_status: 'Download Status',
  home_and_garden_database: 'Home & Garden Database',
  professional_database: 'Professional Database',
  home_and_garden_images: 'Home & Garden images',
  professional_images: 'Professional images',
  media: 'Media',
  content: 'Content',
  sync_offline_data: 'Sync offline data',
  do_you_want_to_start_download: 'Do you want to start download?',
  extracting: 'Extracting',
  detergents: 'Detergents',
  hg: 'HOME & GARDEN',
  pro: 'PROFESSIONAL',
  select_via_machine: 'Selection via machine',
  delete_fav_folder_warning: 'Do you want to delete folder?',
  edit_folder: 'Edit Folder',
  edit_fav_folder_warning: 'Do you want to edit folder name?',
  favorite_folder_edited: 'Favorite folder name edited',
  edit_folder_name: 'EDIT FOLDER NAME',
  duplicate_folder: 'DUPLICATE FOLDER',
  duplicate_fav_folder_warning: 'Do you want to duplicate folder name?',
  favorite_folder_duplicate: 'Favorite folder duplicated',
  cleaning_agents_and_accessories: 'Cleaning agents and accessories',
  please_contact_for_accessories_and_detergents: 'Please contact your dealer for more information on suitable accessories and detergents.',
  major_update_available_alert: 'There is an upgraded version of the App. To work seamlessly need to upgrade the App.',
  patch_update_available_alert: 'There is an upgraded version of the App. Do you want to update the app?',
  export_favorite_as_json: 'Export favorite as Json',
  export_favorite_as_text: 'Export favorite as Text',
  favorite_export_success_alert: 'Favorite exported successfully',
  no_favourite_added_alert: 'Sorry, No favorite is added',
  product_not_available_warning: 'Unfortunately, the desired product is not part of our current product range anymore. Accessories, cleaning products and operating instructions are still available.',
  import_favorites: 'Import Favorites',
  favorite_import_different_country_error: "This file can't be imported because it corresponds to different country. Please switch your country in the settings accordingly",
  favorite_import_different_language_error: 'Favorite data belongs to different Language of the Current Country. While switching to this Country data will be shown.',
  favorite_import_different_invalid_file_error: 'This file is invalid.',
  power_saving_mode_warning: 'Sorry, Power saving mode is on. Please try again.',
  not_part_current_product_range: 'Not part of our current product range',
  enable_analytics_tracking: 'Enable analytics tracking',
  instruction_required: 'This device requires instruction.',
  INFO_NO_RESULTS_FOUND: 'No matches for your search could be found',
  standard_optional_accessories: 'Standard/Optional',
  standard_accessories: 'Standard',
  optional_accessories: 'Optional'
}];
