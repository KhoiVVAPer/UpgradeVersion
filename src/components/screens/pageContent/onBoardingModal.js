/* eslint-disable max-len */
import React from 'react';
import {
  View,
  Text,
  Image,
  Modal
} from 'react-native';
import PropTypes from 'prop-types';
import { Button, HtmlParser } from '../../custom';
import { images, tr } from '../../../config';
import styles from './styles';

const OnBoardingModal = ({
  skipStartPageOnboarding,
  nextStartPageOnboarding,
  translations
}) => (
  <Modal
    transparent
    visible
    onRequestClose={() => { }}
  >
    <View style={styles.step1_OnboardingWrap}>
      <View style={styles.step1_OnboardingInner}>
        <View style={styles.onboardingLogoWrap}>
          <Image
            source={images.logo}
            style={styles.onboardingLogo}
            resizeMode="contain"
          />
        </View>
        <Text style={styles.onboardingHead}>{tr(translations, 'onboarding_root_heading', 'Willkommen im Kärcher Programm')}</Text>
        <HtmlParser
          html={tr(translations, 'onboarding_root_description', 'Lernen Sie die wichtigsten Funktionen im Kärcher Programm kennen.')}
          textKey="onboarding-content-root"
          style={styles.onboardingContent}
        />
        <View style={styles.step1_OnboardingFooter}>
          <Button
            onPress={() => { skipStartPageOnboarding(); }}
            text={tr(translations, 'skip', 'Skip')}
            theme="gray"
          />
          <Button
            onPress={() => { nextStartPageOnboarding(); }}
            text={tr(translations, 'next', 'Next')}
            theme="app"
            style={{ marginLeft: 15 }}
          />
        </View>
      </View>
    </View>
  </Modal>
);

OnBoardingModal.propTypes = {
  skipStartPageOnboarding: PropTypes.func.isRequired,
  nextStartPageOnboarding: PropTypes.func.isRequired,
  translations: PropTypes.arrayOf(PropTypes.any).isRequired
};

export default OnBoardingModal;
