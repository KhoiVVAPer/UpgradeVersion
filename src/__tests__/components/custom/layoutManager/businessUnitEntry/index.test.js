import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import BusinessUnitEntry from '../../../../../components/custom/layoutManager/businessUnitEntry';
import { businessUnitEntry } from '../../../../../assets/data/pageContent';

const setUp = (props) => {
  const component = shallow(<BusinessUnitEntry {...props} />);
  return component;
};

describe('components > custom > layoutManager > BusinessUnitEntry', () => {
  let component;
  const props = {
    layoutData: businessUnitEntry
  };
  beforeEach(() => {
    component = setUp(props);
    return component;
  });

  it('Check snapshot', () => {
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Check if contains renderCatalogue function', () => {
    const componentInstance = component.instance();
    const input = componentInstance.renderCatalogue;
    expect(input).toBeDefined();
  });

  it('Check if contains navigate function', () => {
    const componentInstance = component.instance();
    const input = componentInstance.navigate;
    expect(input).toBeDefined();
  });

  it('Check if renders images properly', () => {
    const input = component.find({ dcaTest: 'image' }).length;
    const output = 2;
    expect(input).toEqual(output);
  });

  it('Check if renders buttons properly', () => {
    const input = component.find('Button').length;
    const output = 2;
    expect(input).toEqual(output);
  });

  it('Check if renders headline with marking properly', () => {
    const input = component.find('HtmlParser').length;
    const output = 2;
    expect(input).toEqual(output);
  });

  it('Check if renders TouchableDebounce for navigation components', () => {
    const input = component.find('TouchableDebounce').length;
    const output = 2;
    expect(input).toEqual(output);
  });
});
