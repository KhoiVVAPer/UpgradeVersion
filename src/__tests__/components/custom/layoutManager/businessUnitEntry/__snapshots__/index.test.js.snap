// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components > custom > layoutManager > BusinessUnitEntry Check snapshot 1`] = `
<react-native-mock>
  <react-native-mock>
    <react-native-mock>
      <react-native-mock>
        <react-native-mock />
      </react-native-mock>
      <react-native-mock />
      <react-native-mock>
        <react-native-mock>
          
        </react-native-mock>
      </react-native-mock>
    </react-native-mock>
  </react-native-mock>
  <react-native-mock>
    <react-native-mock>
      <react-native-mock>
        <react-native-mock />
      </react-native-mock>
      <react-native-mock />
      <react-native-mock>
        <react-native-mock>
          
        </react-native-mock>
      </react-native-mock>
    </react-native-mock>
  </react-native-mock>
</react-native-mock>
`;
